ush/detect_machine.sh

```diff
@@ -8,6 +8,12 @@
 #
 # Thank you for your contribution
 
+# overwrite MACHINE_ID if in container
+if [[ -d /opt/spack-stack && -v SINGULARITY_CONTAINER ]]; then
+  # We are in a container
+  MACHINE_ID=container
+fi
```

Comment:
```suggestion
if [[ -v SINGULARITY_CONTAINER ]]; then
  # We are in a container
  MACHINE_ID=container
fi
```
Isn't `-v SINGULARITY_CONTAINER` enough to know one is in a container?  `/opt/spack-stack` is not a required location for installing the software stack, it is just a suggestion.